examples/most_infeasible_branching.rs

```diff
@@ -1,6 +1,7 @@
-use russcip::prelude::*;
-use russcip::ParamSetting;
-use russcip::Solving;
+use russcip::{
+    prelude::*, BranchRule, BranchingCandidate, BranchingResult, ParamSetting, SCIPBranchRule,
```

Comment:
I had errors displayed in my editor on missing uses, but it compiled anyway. I added them to get rid of the warnings.